peer/hashring32/ring.go

```diff
@@ -156,6 +163,10 @@ func (pr *peerRing) Choose(req *transport.Request) peer.StatusPeer {
 		}
 	}
 
+	if !ok && pr.offsetGeneratorValue != 0 {
```

Comment:
Nit: this line could be `else if pr.offsetGeneratorValue != 0` in the previous if block?